client/src/test/java/com/kritsit/casetracker/client/domain/services/EditorTest.java

```diff
@@ -37,7 +37,7 @@ public void setUp() {
     }
 
     public void testCreation() {
-        assertTrue(editor instanceof IEditorService);
+        assertNotNull(editor);
```

Comment:
"instanceof will always return true"

"This instanceof test will always return true (unless the value being tested is null). Although this is safe, make sure it isn't an indication of some misunderstanding or some other logic error. If you really want to test the value for being null, perhaps it would be clearer to do better to do a null test rather than an instanceof test."

Apparently the only thing worth check is that it isn't null.